makefile

```
@@ -61,18 +74,39 @@ gen-proto:
 
 	cp -R internal/proto/github.com/open-telemetry/opamp-go/protobufs/* protobufs/
 	rm -rf internal/proto/github.com/
+	$(MAKE) fmt
 
 .PHONY: gomoddownload
 gomoddownload:
-	go mod download
+	$(GOCMD) mod download
 
 .PHONY: install-tools
-install-tools:
-	cd $(TOOLS_MOD_DIR) && go install github.com/ory/go-acc
+install-tools: $(TOOLS_BIN_NAMES)
+
+$(TOOLS_BIN_DIR):
+	mkdir -p $@
+
+$(TOOLS_BIN_NAMES): $(TOOLS_BIN_DIR) $(TOOLS_MOD_DIR)/go.mod
+	cd $(TOOLS_MOD_DIR) && $(GOCMD) build -o $@ -trimpath $(filter %/$(notdir $@),$(TOOLS_PKG_NAMES))
 
 .PHONY: tidy
 tidy:
 	rm -fr go.sum
-	go mod tidy
-	cd internal/examples && rm -fr go.sum && go mod tidy
-	cd internal/tools && rm -fr go.sum && go mod tidy
+	$(GOCMD) mod tidy
+	cd internal/examples && rm -fr go.sum && $(GOCMD) mod tidy
+	cd $(TOOLS_MOD_DIR) && rm -fr go.sum && $(GOCMD) mod tidy
+
+.PHONY: fmt
+fmt: common/gofmt common/goimports common/gofumpt
```

This looks unnecessarily complicated. Do we ever need to run them separately? Each subtarget runs just a single command, so perhaps simply list the 3 commands here and eliminate the subtargets:
```suggestion
fmt:
  gofmt -w -s ./
  $(GOIMPORTS) -w -local go.opentelemetry.io/collector ./
  $(GOFUMPT) -l -w .
```
These are actually supposed to be run sequentially so making them separate targets is strictly speaking incorrect (and will likely break with `make -j` when they try to concurrently modify the same file).

---

You are right, in most cases we end up just running the `fmt` target. Let's unify them for now https://github.com/open-telemetry/opamp-go/pull/356/commits/eed6e204d026cbba650126f92e3c0b978e5e2072